perl-package/AI-MXNetCAPI/mxnet.i

```diff
@@ -1633,6 +1633,8 @@ int MXOptimizeForBackend(SymbolHandle sym_handle,
                                    const mx_uint in,
                                    NDArrayHandle* in,
                                    const mx_uint in,
+                                   NDArrayHandle* in,
```

Comment:
should we have the same names for this function params?

Reply:
I was just following the style that was already there. @sergeykolychev to verify